tests/network/vmi_multus.go

```diff
@@ -1022,6 +1022,59 @@ var _ = Describe("[Serial]SRIOV", func() {
 				expectInterfaceToExistByMac(virtClient, vmi, mac, hotplugSRIOVDeviceTimeout,
 					"SR-IOV VF is expected to exist in the guest after migration")
 			})
+
+			Context("when aborted", func() {
+				const migrationRunningTimeout = 180
+				const migrationCompleteTimeout = 180
+
+				const targetPodDisposalTimeout = 120
+
+				const vmiMemoryStressSize = 200
+				const vmiStressTimeout = 600
+
+				var migration *v1.VirtualMachineInstanceMigration
+				var migrationUID string
+
+				BeforeEach(func() {
+					// slow down migration in order to assert on migration phase
+					// running by putting pressure on VMI memory
+					tests.RunStressTest(vmi, vmiMemoryStressSize, vmiStressTimeout)
+
+					By("Starting a migration")
+					migration = tests.NewRandomMigration(vmi.Name, vmi.Namespace)
+					migration, err = virtClient.VirtualMachineInstanceMigration(migration.Namespace).Create(migration)
+					migrationUID = string(migration.UID)
+
+					tests.ExpectMigrationPhaseRunning(virtClient, migration, vmi, migrationRunningTimeout)
+				})
+
+				It("should re-attach SRIOV host-devices to the source VM when failed", func() {
+					updatedVMI, err := virtClient.VirtualMachineInstance(vmi.Namespace).Get(vmi.Name, &metav1.GetOptions{})
```

Comment:
nit: In the other tests, `vmi` was expected to be refreshed to represent the latest status. Why we do this differently now?
Once the "wait" for the phases finishes, it makes sense for it to return with an updated vmi as well IMO.
Anyway, the main reason I even noticed it is because the name here of the vmi variable changed and I was trying to understand why there is a need for two (e.g. do we compare the old and the new?).

Not a must.

Reply:
I added this VMI fetch on this test before adding the annotation because the `Update` command fails because the VMI version is not the latest.

Looking at this again I dont think we need another variable for the update VMI, I will reuse the one that already there.